x-pack/plugins/apm/public/components/app/service_map/Popover/backend_contents.tsx

```diff
@@ -12,12 +12,20 @@ import { METRIC_TYPE } from '@kbn/analytics';
 import React from 'react';
 import { useUiTracker } from '../../../../../../observability/public';
 import { ContentsProps } from '.';
-import { NodeStats } from '../../../../../common/service_map';
 import { useAnyOfApmParams } from '../../../../hooks/use_apm_params';
 import { useApmRouter } from '../../../../hooks/use_apm_router';
 import { FETCH_STATUS, useFetcher } from '../../../../hooks/use_fetcher';
 import { ApmRoutes } from '../../../routing/apm_route_config';
 import { StatsList } from './stats_list';
+import { getTimeRangeComparison } from '../../../shared/time_comparison/get_time_range_comparison';
+import { APIReturnType } from '../../../../services/rest/createCallApmApi';
+
+type BackendReturn = APIReturnType<'GET /internal/apm/service-map/backend'>;
+
+const INITIAL_STATE: BackendReturn = {
+  currentPeriod: {},
+  previousPeriod: undefined,
+};
```

Comment:
I've been thinking a bit about this "initial state". Do we still need this, now when we have optional chaining that makes it very easy to traverse an object with optional properties. Eg `data?.currentPeriod?.foo.bar`